src/store/actions.ts

```diff
@@ -205,10 +205,15 @@ export default {
   }),
   save: tryAction(async ({ state, commit }: {state: ApplicationState, commit: any}) => {
     const formFields = [...state.orderFormFields, { id: 'contents', type: 'text' }]
+    // @ts-ignore
+    const { context: { email, username } } = state.context
+
     const formData = {
       ...state.order,
-      ...{ contents: JSON.stringify(toCart(state)) },
-      ...{ updateDate: moment().toISOString() }
+      contents: JSON.stringify(toCart(state)),
+      updateDate: moment().toISOString(),
+      email,
```

Comment:
same thing needs to happen to submit action of refactor to use shared part

Reply:
Added same functionality to submit.